docs/source/guide/dataset_create.md

```diff
@@ -20,26 +20,217 @@ hide_sidebar: true
+
+    ```json
+
+           "Resource": [
+                "arn:aws:s3:::<your_bucket_name>",
+                "arn:aws:s3:::<your_bucket_name>/*"
+            ]
+
+    ```
+
+
+###### Create a user 
+
+This user can be tied to a specific person or a group. 
+
+1. From the AWS Management Console, use the search bar or navigation menu to locate the **IAM** service.
+
+2. Select **Access Management > Users** from the menu on the left. 
+
+3. Click **Create user**.
+
+4. Enter a descriptive name for this user, such as “Label_Studio_access”. 
+
+    Leave **Provide user access to the AWS Management Console** unselected. Click **Next**.
+
+5. Select **Attach policies directly**.
+
+6. Under **Permissions policies**, use the search field to find and select the policy you are using with the user (see above). Click **Next**.
+
+7. Click **Create user**.
+
+
+
+###### Generate an access key for the user
+
+1. From the **Users** page, click the user you created in the previous section.  
+
+2. Click the **Security Credentials** tab.
+
+    ![Screenshot of the Security Credentials option](/images/data_discovery/aws_key.png)
+
+3. Scroll down to **Access keys** and click **Create access key**.
+4. Select **Other** and note the recommendations provided by AWS. Click **Next**.
+5. Optionally, add a description for the key.
+6. Click **Create access key**.
+7. Copy the access key ID and your secret access key and keep them somewhere safe, or export the key to a CSV file.
+
+    ![Screenshot of the copy icon next to the key](/images/data_discovery/aws_key2.png)
+
+    <div class="admonition attention"><p class="admonition-title">Important</p><p>This is the only time you will be able to copy the secret access key. Once you click <strong>Done</strong>, you will not be able to view or copy it again.</p></div>
+
+8. Click **Done**. 
+   
+{% enddetails %}
+
+### Create a dataset from an AWS S3 bucket
+
+1. From Label Studio, navigate to the Datasets page and click **Create Dataset**. 
+
+    ![Create a dataset action](/images/data_discovery/dataset_create.png)
+
+2. Complete the following fields and then click **Next**:
+
+    <div class="noheader rowheader">
+
+    | | |
+    | --- | --- |
+    | Name | Enter a name for the dataset. |
+    | Description | Enter a brief description for the dataset.  |
+    | Source | Select AWS S3. |
+
+    </div>
+
+3. Complete the following fields: 
+
+    <div class="noheader rowheader">
+
+    | | |
+    | --- | --- |
+    | Bucket Name | Enter the name of the AWS S3 bucket. |
+    | Bucket Prefix | Enter the folder name within the bucket that you would like to use.  For example, `data-set-1` or `data-set-1/subfolder-2`.  |
+    | File Name Filter | Use glob format to filter which file types to sync. For example, to sync all JPG files, enter `*jpg`. To sync all JPG and PNG files, enter `**/**+(jpg\|png)`.<br><br>At this time, we support the following file types: .jpg, .jpeg, .png, .txt, .text |
```

Comment:
Maybe I'm missing something obvious, but why does jpg/png require a more complex prefix than just jpg? I'd expect it to just be `*(jpg|png)`?

Reply:
I think that's just the way glob works? `*(jpg|png)` wouldn't work -- see https://globster.xyz/. FWIW, I was trying to give a somewhat simpler example vs. what we give in the placeholder, which is `**/**+(jp?(e)g|png)`

Reply:
Updated with your suggestion.